### DISCUSSION TITLE
Support dataSource for PVC in PersistentClaimStorage

### DISCUSSION THREAD

**Discussion by jbpin** [strimzi/strimzi-kafka-operator#7137] [Ideas]
Since kubernetes v1.20.0 it's possible to restore a PVC from a [Snapshot](https://kubernetes.io/docs/concepts/storage/volume-snapshots/). 

As recovery strategy It could be interesting to be able to configure the `dataSource` for PVC in the [PersistentClaimStorage](https://strimzi.io/docs/operators/latest/configuring.html#type-PersistentClaimStorage-reference)

**Comment by scholzj** [comment]
I guess you can probably follow the procedure for recovering the cluster from PVC and use the snapshots there.

**Reply by jbpin** [reply]
I guess. Please, do you have any link to this procedure? I can't find it on the documentation website

**Reply by scholzj** [reply]
https://strimzi.io/docs/operators/latest/full/configuring.html#cluster-recovery-str

**Reply by jbpin** [reply]
Thanks. I'll try
